controllers/adminController.js

```diff
@@ -40,6 +40,28 @@ const controller = {
       res.status(400).json(error.message);
     }
   },
+
+  //forget
+  forget: async (req, res) => {
+    try {
+      const { email, password } = req.body;
+      if (!email || !password) throw Error("All fields are mandatory");
+      const { rows } = await pool.query(
+        "SELECT * FROM admin WHERE email = $1",
```

Comment:
Please add a condition to check password as well, this way you are not authenticating the user